src/MinHeap.js

```diff
@@ -0,0 +1,104 @@
+var MinHeap = function() {
+  var values = [null];
```

Comment:
Not related to the implementation, but any reason not to use `const` here?

Reply:
@visola I decided to write everything in `ES5` - technically, `const` and `let` do not exist in `ES5` JS